src/routes.js

```diff
@@ -15,8 +17,10 @@ export default () => (
     <Router>
       <Switch>
         <Route exact path="/" component={Home} />
-        <Route exact path="/signin" component={Signin} />
+        <Route path="/article/:id" component={Read} />
+        <Route exact path="/articles/new" component={Create} />
         <Route exact path="/login" component={SocialAuth} />
+        <Route exact path="/signin" component={Signin} />
```

Comment:
With the recent dev changes, we have a central Route point. Let's use it for our routes

Reply:
Okay making the changes.